### ISSUE TITLE
DFormat executable path is not valid even though dfmt is in /usr/bin

### ISSUE THREAD

**Issue by HossainAdnan** [intellij-dlanguage/intellij-dlanguage#519]
Even though dfmt is installed in /usr/bin, triggering it with Ctrl + Alt + K triggers the error message stated above. 
```
$ dfmt --version
v0.10.1
```

Events log:
```
28/9/19
2:00 pm	Reformat code with DFormat
				DFormat executable path is not valid
				Configure

2:07 pm	Reformat code with DFormat
				DFormat executable path is not valid
				Configure

2:07 pm	Reformat code with DFormat
				DFormat executable path is not valid
```

![image](https://i.imgur.com/4Yg20MN.png)

**Comment by andre2007** [comment]
@HossainAdnan please check that full path to dfmt is entered in tabstrip DFormat. If you use button "Auto Find" in addition I had the issue that there was space suffix in the edit field causing issues.

**Comment by HossainAdnan** [comment]
Can confirm, the issue was the space suffix, manually trimming the path fixed it.

**Comment by SingingBush** [comment]
We should trim the path in the code base then.

**Comment by andre2007** [comment]
@SingingBush there are actually 2 bugs.

- the path should be trimmed
- "auto find" proposes a path with an additional  space suffix

**Comment by SingingBush** [comment]
This should be fixed now as part of #514 

Can you pull the changes in develop and run it locally: `./gradlew runIde`

I'll get a new release out soon

**Comment by SingingBush** [comment]
Closing as this has been fixed already and due out in next release
